test/test_cardgames.py

```diff
@@ -0,0 +1,23 @@
+from testing_base import *
+
```

Comment:
test_cardgames.py is not super descriptive of what this is testing, you could name it something like test_printCards.py or something along those lines.

Reply:
Thanks, I adjusted my code based on all your changes, I believe they are included in my next commits, but there is one file that I am still trying to upload...my bash terminal thinks it has been deleted so I am working that out